tests/sles4sap/nw_ascs_install.pm

```diff
@@ -64,10 +64,10 @@ sub run {
 
     # Check everything was copied correctly
     my $cmd = q|find . -type f -exec md5sum {} \; > /tmp/check-nw-media|;
-    assert_script_run $cmd, 300;
+    assert_script_run $cmd, 600;
     type_string "cd /sapinst\n";
     assert_script_run "umount /mnt";
-    assert_script_run "md5sum -c /tmp/check-nw-media", 300;
+    assert_script_run "md5sum -c /tmp/check-nw-media", 600;
```

Comment:
Maybe you could use a variable for the '600' value, as you use it multiple times. Could be helpful if another timing tweak is needed in a the future ;-)

Other than this, LGTM!

Reply:
Good idea. Will do that